pkg/alertmanager/api_test.go

```diff
@@ -125,7 +123,7 @@ template_files:
 	}
 
 	am := &MultitenantAlertmanager{
-		store:  noopAlertStore{},
+		store:  prepareFilesystemAlertStore(t),
```

Comment:
Better to use an empty real storage than a mocked one.

Reply:
We currently don't allow to configure the in-memory bucket via the `bucket.Config`. Instead of doing changes to use it, I just used the filesystem-based one which is few lines more of code because of the temporarily directory.

Reply:
We don't need to use `bucket.Config`, do we?

Reply:
```
func prepareFilesystemAlertStore(t *testing.T) alertstore.AlertStore {
	obj := objstore.NewInMemBucket()
	return bucketclient.NewBucketAlertStore(obj, nil, log.NewNopLogger())
}
```

Reply:
Yeah, done. I was smoking some good weed when I first replied you.